frameworks/D/kiss/benchmark_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "framework": "kiss",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
```

Comment:
FYI, since kiss does not appear to be a web application framework and since it doesn't appear to provide any functionality related to HTTP, we're going to change this to `"approach": "Stripped"`.  It will be hidden on the results website by default.

Reply:
That's it. The [Collie](https://github.com/huntlabs/collie) is base on Kiss.